### DISCUSSION TITLE
Use Orchard Core to build modular application on ASP.NET Core with Identity Framework

### DISCUSSION THREAD

**Discussion by Jin-K-Yang** [OrchardCMS/OrchardCore#10287] [Q&A]
I want to use Orchard Core to build a modular application, one of the modular application is using Identity Framework.

https://github.com/OrchardCMS/OrchardCore.Samples

I modify the sample code from Orchard's website. I can access the route "modular_project_name/Home/Index" from another project, but I can't access the route "modular_project_name/Identity/Account/Login" (which is a login view in Identity Framework).

Is there any suggestion for this issue? If there is please help me. Thank you very much!

**Comment by ns8482e** [comment]
RCL `Area` is not working with OrchardCore so I guess Identity UI is also not supported.
Related issue #4247
// cc  @jtkech

**Comment by sebastienros** [comment]
> I can't access the route "modular_project_name/Identity/Account/Login" 

What are you seeing?

**Comment by Jin-K-Yang** [comment]
It can't found the page, it's a 404 not found.
